### ISSUE TITLE
Project Feedback!

### ISSUE THREAD

**Issue by codepathreview** [Demba3/Twitter-Clone#2]
It looks like the following features are not reflected on your GIF walkthrough: 

- User is displayed the relative timestamp for each tweet "8m", "7h"

Since the extension period is over, we will no longer regrade any resubmissions made after this date. We just want to let you know so that you make sure to reflect the features on your GIF for future submissions

**Comment by Demba3** [comment]
Yes, you are right. Sorry about that, it was a mistake. But I do not understand why I scored a 6 out of 10. If that is the only feature I am missing and it is a sub feature of a feature worth 4 points.

**Comment by codepathreview** [comment]
Hello there! 
We have reviewed your scores and here are details of some features partially shown in your GIF:
- User can sign in to Twitter using OAuth login.
    -   _This feature is only given 1 point because the authentication screen is not shown. For this feature, the authentication part when logging in must be shown to be considered fully implemented and given full scores._ 

- User is displayed the username, name, and body for each tweet
   - _This feature is also considered partially implemented because the tweets do not show the name of the person tweeting, rather only the body of each tweet and the username are shown._
